arangod/Agency/Job.cpp

```diff
@@ -381,9 +371,9 @@ std::string Job::findNonblockedCommonHealthyInSyncFollower( // Which is in "GOOD
 
 }
 
-///
-/// ShortName comes "later" in server start sequence.  Could it be missing
-///  at this point, and would that cause a throw?
+
+// Lookup a shortID.
+// If the server does not have a shortName yet, return empty string.
 std::string Job::uuidLookup (std::string const& shortID) {
   for (auto const& uuid : _snapshot(mapUniqueToShortID).children()) {
     if ((*uuid.second)("ShortName").getString() == shortID) {
```

Comment:
My comment was about this line.  I believe a server could exist yet the short name have not arrived.  That implies this could throw.  We saw that one other place.  At this point, looking for you to agree or disagree ... not necessarily via comment in code.  I would pick this up in the Node.cpp cleanup.

Reply:
My bad. I was sure that there was no way for existing in `Plan/DBservers` or `Plan/Coordinators` without `MapUniqueToShortID`. mop rewrote that part of the code that removed that necessity. I thereby agree.